### PR TITLE
Fixed #57

### LINKED ISSUE #57: Problem with floating point scalars
When i try to define floating point numbers in yaml files type provider reports following error:
'FSharp.Configuration.ConfigTypeProvider+FSharpConfigurationProvider' reported an error: Input string was not in a correct format.	

Example yaml file (issue reproduces):
```yaml
Test:
    AFloatingPointNumber: 0.1
```

This works fine:
```yaml
Test:
    AIntegerNumber: 1
```

### PR DESCRIPTION
**Pull request by Turee**
- Updated SharpYaml.dll
- Use YamlParsers results instead of casting objects to string and then
parsing them back to original types

### CODE REVIEW

File: src/FSharp.Configuration/YamlConfigProvider.fs [hunk 1]
```diff
@@ -22,12 +22,21 @@ module private Parser =
...
         | Bool of bool
         | Uri of Uri
-        static member Parse = function
-            | ValueParser.Bool x -> Bool x
-            | ValueParser.Int x -> Int x
+        | Float of double
+        static member ParseStr = function
+            //| ValueParser.Bool x -> Bool x
+            //| ValueParser.Int x -> Int x
+            //| ValueParser.Float x -> Float x
```
Comment (vasily-kirichenko) [inline src/FSharp.Configuration/YamlConfigProvider.fs]: Could you remove the commented code?

File: tests/FSharp.Configuration.Tests/Settings.yaml [hunk 2]
```diff
@@ -9,11 +9,13 @@
     Port: 331
     User: user2
     Password: pass2
-    CheckPeriod: 00:01:00
+    CheckPeriod: 00:01:00.0000000
```
Comment (vasily-kirichenko) [inline tests/FSharp.Configuration.Tests/Settings.yaml]: So short `TimeSpan`s are not parsed now?

Comment (Turee) [inline tests/FSharp.Configuration.Tests/Settings.yaml]: Yes they are as tested in 
```F#
[<Test>] 
let ``Can loads full settings``() =
....
```

File: src/FSharp.Configuration/YamlConfigProvider.fs (hunk 1 above)
Comment (Turee) [inline src/FSharp.Configuration/YamlConfigProvider.fs]: Removed comments as requested, cleaned up code a bit too

**Comment by vasily-kirichenko** [comment]
Thanks! I've just published the NuGet package.

**Comment by Turee** [comment]
Thank you ! :)

**Pull request merged**